### PR TITLE
Enable the Rails/NotNullColumn Rubocop cop

### PR DESCRIPTION
**Pull request by mschnitzer**
I'm not sure if this is the correct way to implement this. Would appreciate some feedback.

### CODE REVIEW

**Comment by adrianschroeter** [comment]
well, you can change existing migrations, you just can't count that they will re-run ....
(we have to sometimes because the migration is failing for people)

however, it is a bit risky, since we can't test them as good as other parts and we don't have that many production data. so you must be really super confident that it does not break anything ....

**Comment by adrianschroeter** [comment]
but you need to verify that this must not change the resulting structure.sql ... did you?

**Comment by evanrolfe** [comment]
@adrianschroeter I don't think we can verify that these new migrations result in the same structure.sql until this card is finished? or?
https://trello.com/c/inrAJxQq/176-8-evaluate-and-fix-missing-migrations

**Comment by hennevogel** [comment]
> well, you can change existing migrations, you just can't count that they will re-run ....

http://guides.rubyonrails.org/active_record_migrations.html#changing-existing-migrations

**Comment by adrianschroeter** [comment]
@evanrolfe you could rollback the migration and run it again as smoke test, there shouldn't be a diff for structure.sql ...

but that does is no guarantee that we won't have a regression with some real life production data...

**Comment by mschnitzer** [comment]
I like the idea of @hennevogel: Can't we just skip the old migrations and just enable the cop so that this rule applies for newly created migrations?

**Comment by mschnitzer** [comment]
@DavidKang @Ana06 @hennevogel please review again, I updated the commit.

File: src/api/.rubocop.yml [hunk 1]
```diff
@@ -244,6 +244,15 @@ Rails/Output:
 Rails/Validation:
   Enabled: true
 
+# Requires each table column to have a default value
+Rails/NotNullColumn:
```
Comment (Ana06) [inline src/api/.rubocop.yml]: Order this alphabetically!

**Comment by mschnitzer** [comment]
@Ana06 done

**Pull request merged**